docker/main/ngen/constraints.txt

```diff
@@ -0,0 +1,5 @@
+### Constrain pip
+pip>=23.0,<23.1
```

Comment:
I don't think this is necessary after https://github.com/NOAA-OWP/t-route/pull/804 was merged.

Reply:
Doesn't have to be addressed here, just putting it on your radar @robertbartel!